README.md

```diff
@@ -1,2 +1,104 @@
 # learn-raspberry-pi-iot
 Learn how to build Internet of Things (IoT) Apps using Raspberry Pi and JavaScript!
+
+<img src="https://www.raspberrypi.org/wp-content/uploads/2012/03/raspberry-pi-logo.png" width="40%" style="display: block; margin: 0 auto;">
+
+## *Why*?
+
+A **[Raspberry Pi](https://vimeo.com/90103691)** (Raspberry Pi) is a 'mini computer' that was created to help people learn about the basics of computers. They are _great_ for resource to get started on learning about hardware as they consist of the basic components of a computer. Raspberry Pi can run on an OS, which means that we can talk to the computer with higher level languages like Javascript!
```

Comment:
This paragraph is _good_.
But please _avoid_ super-long lines of text.
please read: http://rhodesmill.org/brandon/2012/one-sentence-per-line